commands/netobserv

```diff
@@ -228,28 +241,22 @@ if [[ "$command" == "flows" || "$command" == "packets" ]]; then
       -n $namespace \
       collector \
       -- $execCommand
-  else
+  elif [[ "$command" == "flows" || "$command" == "packets" ]]; then
     echo "Background capture started. Use:"
     echo " - '${K8S_CLI_BIN} netobserv follow' to see the capture progress"
     echo " - '${K8S_CLI_BIN} netobserv copy' to copy the generated files locally"
     echo " - '${K8S_CLI_BIN} netobserv cleanup' to remove the netobserv components"
-  fi
-elif [ "$command" = "metrics" ]; then
-  if [[ "$outputYAML" == "true" ]]; then
-    echo "Check the generated YAML file in output folder."
     echo
-    echo "You can create metrics agents by executing:"
-    echo " ${K8S_CLI_BIN} apply -f ./output/${command}_capture_${dateName}.yml"
-    echo 
-    echo "Then open your OCP Console and search for netobserv-cli dashboard"
+    echo "Capture end automatically in $maxTime or reaching $maxBytes bytes"
+  else
+    runBackground="true"
+    echo "Metrics capture started."
+    consoleURL="$(oc whoami --show-console)"
+    echo "Open ${consoleURL}/monitoring/dashboards/netobserv-cli to see generated metrics."
+    echo "Use 'oc netobserv stop' to stop the collection and 'oc netobserv cleanup' to remove everything."
```

Comment:
shouldn't it be `${K8S_CLI_BIN}` used instead of `oc` ?

Reply:
(unless that's because it's an openshift-only feature..)

Reply:
https://github.com/netobserv/network-observability-cli/pull/320/commits/5b646d21826f6c4ee75b27c20c504bf8ee71bc95